test/test_torch.py

```diff
@@ -11963,6 +11963,18 @@ def test_memory_format_preserved_after_permute(self, device):
         y = nhwc.permute(0, 1, 3, 2).permute(0, 1, 3, 2)
         self.assertTrue(y.is_contiguous(memory_format=torch.channels_last))
 
+    def test_resize_as_preserves_strides(self, device):
+        x = torch.empty(2, 3).t()
+        old_strides = x.stride()
+        x.resize_as_(x)
+        self.assertEqual(x.stride(), old_strides)
+
+    def test_memory_format_resize_as(self, device):
```

Comment:
we should have tests for all the cases, i.e. add a:
1) explicit contiguous test
2) explicit channels_last test.